rpy2/tests/rinterface/test_embedded_r.py

```diff
@@ -73,12 +78,12 @@ def test_set_initoptions_after_init():
 
 
 def test_initr():
-    def init_r():
-        from rpy2 import rinterface
-        rinterface.initr()
     preserve_hash = True
-    proc = multiprocessing.Process(target=init_r,
-                                   args=(preserve_hash,))
+    args = ()
```

Comment:
It seems that `test_initr()` has no longer a purpose because of code drift. I have opened issue #691 to follow-up on this.